operations/package_repository_management/Nexus_Export_and_Restore.md

```diff
@@ -85,3 +85,22 @@ If the old export is not deleted, then the new job will overwrite the old export
 ```bash
 kubectl delete pvc -n nexus nexus-bak
 ```
+
+### Cleanup failed or stopped export
+
+If an export is stopped prematurely or fails to complete there is a few steps that need to be taken to bring Nexus back into a working state.
+
+1. Delete the failed or stopped job, see [Cleanup export job](#cleanup-export-job)
+1. If the export was stopped or failed delete the partially filled export PVC, see [Cleanup previous export](#cleanup-previous-export)
```

Comment:
should this be "failed TO delete the partially..."?

Reply:
It should drop the first half of that sentence as its redundant.